### PR TITLE
[enable counters] provide initial rates parameters

### PR DESCRIPTION
**Pull request by mykolaf**
Signed-off-by: Mykola Faryma <mykolaf@mellanox.com>

**- Why I did it**
The lua plugins `port_rates.lua` and `rif_rates.lua` rely on the parameter of alpha. We should provide some a default value for this.
Currently the scripts flood with error because of missing value from db.
**- How I did it**

**- How to verify it**
Run the lua script, finishes with no error.
**- Which release branch to backport (provide reason below if seleted)**

- [ ] 201811
- [ ] 201911
- [ ] 202006

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: dockers/docker-orchagent/enable_counters.py [hunk 1]
```diff
@@ -3,11 +3,24 @@
 import swsssdk
 import time
 
+DEFAULT_SMOOTH_INTERVAL = '10'
```
Comment (liat-grozovik) [inline dockers/docker-orchagent/enable_counters.py:6]: can you explain why you decided to use these default values?
are those values used for other counters?

Comment (lguohan) [reply dockers/docker-orchagent/enable_counters.py:6]: any update on this question?

Comment (mykolaf) [reply dockers/docker-orchagent/enable_counters.py:6]: RIF/PORT_ALPHA is only only used for rates calculation. ALPHA is calculated based on the smoothing interval which defines the size of the window over which we calculate the average value. In this case I configured the window to be 10s. This way if we have a huge 1s spike in traffic, the average rate value will show a curve descending from the spike to the usual rate over approximately 10s.

**Comment by akokhan** [comment]
@lguohan , @liat-grozovik , please approve and merge.

**Review by liat-grozovik — DISMISSED** [review]

**Comment by akokhan** [comment]
retest vsimage please

**Review by lguohan — DISMISSED** [review]

**Comment by lguohan** [comment]
can you resolve the conflict?

**Comment by liat-grozovik** [comment]
@volodymyrsamotiy can you please handle the conflicts on behalf of mykola so we can forward with the merge?

**Comment by vboykox** [comment]
retest broadcom please

**Comment by liat-grozovik** [comment]
retest broadcom please

**Review by liat-grozovik — DISMISSED** [review]

**Review by lguohan — APPROVED** [review]

**Comment by lguohan** [comment]
is there a dependency between this pr and the pr in sonic utilities?

https://github.com/Azure/sonic-utilities/pull/817

**Review by liat-grozovik — APPROVED** [review]

**Comment by liat-grozovik** [comment]
> is there a dependency between this pr and the pr in sonic utilities?
> 
> [Azure/sonic-utilities#817](https://github.com/Azure/sonic-utilities/pull/817)

The dependency are on merged PRs
https://github.com/Azure/sonic-swss-common/pull/330
https://github.com/Azure/sonic-sairedis/pull/562
https://github.com/Azure/sonic-swss/pull/1201

**Comment by zhaoxiaoke** [comment]
test

**Pull request merged**